src/CSMCommon_Consolidate.jl

```diff
@@ -276,8 +277,11 @@ function sendCurrentUpMsg_StateMachine(csmc::CliqStateMachineContainer)
   prepPutCliqueStatusMsgUp!(csmc, upmsg=upmsg)
 
   # also send a down message -- seem weird while doing #459 but okay
-  cliqst = getCliqueStatus(csmc.cliq)
-  notifyCliqDownInitStatus!(csmc.cliq, cliqst, logger=csmc.logger)
+  #XXX #TODO Change NOT Covered in IIF or RoME tests!
+  # cliqst = getCliqueStatus(csmc.cliq)
+  # notifyCliqDownInitStatus!(csmc.cliq, cliqst, logger=csmc.logger)
+  cliqst = prepPutCliqueStatusMsgDwn!(csmc)
+  @warn("XXX Checking notifyCliqDownInitStatus! coverage", c=csmc.cliqKey, cliqst=cliqst)
```

Comment:
Ok, I merge the newest master into this and the test now hits these lines. However, the solve stalled and then errored with:
```julia
┌ Warning: XXX Checking notifyCliqDownInitStatus! coverage
│   c = 2
│   cliqst = :initialized
└ @ IncrementalInference ~/.julia/dev/IncrementalInference/src/CSMCommon_Consolidate.jl:284
(idx, sum(accmSolDims)) = (1, -1.0)
tdims[idx] = sum(accmSolDims) = 0.0
(idx, sum(accmSolDims)) = (2, 0.0)
tdims[idx] = sum(accmSolDims) = 0.0
(idx, sum(accmSolDims)) = (1, 1.0)
tdims[idx] = sum(accmSolDims) = 1.0
(idx, sum(accmSolDims)) = (2, 2.0)
tdims[idx] = sum(accmSolDims) = 0.0
(idx, sum(accmSolDims)) = (3, 1.0)
tdims[idx] = sum(accmSolDims) = 1.0
(idx, sum(accmSolDims)) = (1, -1.0)
tdims[idx] = sum(accmSolDims) = 0.0
(idx, sum(accmSolDims)) = (2, 0.0)
tdims[idx] = sum(accmSolDims) = 0.0



InterruptException:InterruptException:InterruptException...
```

Reply:
Also now triggered and successfully completes in `@testset "test endless cycle case, issue #754"`.